extensions/3DTILES_bounding_volume_S2/0.0.0/README.md

```diff
@@ -1,75 +1,92 @@
 
-This extension to 3D Tiles enables using S2 cells as a `boundingVolume`, and uses `3DTILES_implict_tiling` to enforce subdivision.
+|Level 0|Level 1|
+|:-:|:-:|
+| ![Plane - Level 0](figures/plane_root.png) S2 cell (`"1"`) on the Earth cube| ![Plane - Level 1](figures/plane_children.png) Children of S2 cell (`"1"`) on the Earth cube |
+| ![Ellipsoid - Level 0](figures/ellipsoid_root.png)  S2 cell (`"1"`) on the WGS84 ellipsoid| ![Ellipsoid - Level 1](figures/ellipsoid_children.png) Children of S2 cell (`"1"`) on the WGS84 ellipsoid|
 
-| S2 Curve on Cube Face  |  S2 Curve on WGS84 Ellipsoid |
-|---|---|
+In S2, each face of the unit cube can be subdivided into 30 levels using a quadtree structure, in which each "cell" or tile on the grid subdivides into 4 equal cells or tiles at the subsequent level. This extension to 3D Tiles enables using these cells as the basis for bounding volumes for tiles.
+
+| S2 Curve on Earth cube |  S2 Curve on WGS84 ellipsoid |
+|:-:|:-:|
 | ![Math](figures/plane.png)  | ![Math](figures/ellipsoid.png)  |
 
+The S2 library uses a modified Hilbert curve to provide a one dimensional ordering of cells on the S2 Earth cube. This provides each cell, from level 1 to level 30, with a unique 64-bit identifier. Therefore, we can uniquely identify millimeter scale areas on Earth with their S2 cell IDs.
+
 ## Coordinate System
 
-The S2 library does not mandate the usage of geocentric or geodetic coordinates.This extension uses WGS84 geodetic coordinates for mapping the points between the Earth and the S2 sphere.
+The S2 library does not mandate the usage of geocentric or geodetic coordinates. This extension uses WGS84 geodetic coordinates for mapping the points between the Earth and the S2 sphere.
+
+## Bounding Volume
+
+This extension enables using S2 cells and a set of bounding heights to describe a bounding volume in 3D Tiles. The `token` represents an S2 cell. The cell describes 4 vertices on the surface of the WGS84 ellipsoid for the area, and the `minimumHeight` and `maximumHeight` describe the vertical limits with respect to the WGS84 ellipsoid.
 
-## Cell Token
+![Volume](figure/../figures/volume.jpg)
 
-This extension uses tokens, or hexadecimal string representations of `S2CellId` for two reasons:
+<center>S2 cell (<tt>"89c6c7"</tt>) covering the Philadelphia Center City area, with minimum height set to <tt>0</tt> m and maximum height set to <tt>1000</tt> m.</center>
+
+### Cell Token
+
+This extension uses tokens, or hexadecimal string representations of S2 cell identifier for two reasons:
  1. Precision: Using a token will require a client to convert it to the correct data type: `uint64`
- 2. Conciseness: Tokens provide a more concise representation of `S2CellId`s
+ 2. Conciseness: Tokens provide a more concise representation of identifiers
```

Comment:
Could you explain what you mean by "more concise representation"? Not sure I'm following.

Reply:
I added an example to show, not just tell:
>  For example, the root (level 0) cell with cell ID `3458764513820540928` has the token `"3"`.